<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Judger</class>
 <widget class="QWidget" name="Judger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1387</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>0</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Human Judger</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Commit">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>680</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Commit</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="File_A_Code">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>561</width>
     <height>551</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="File_B_Code">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>100</y>
     <width>531</width>
     <height>551</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="File_A_Name">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="File_B_Name">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>60</y>
     <width>531</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Quit">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>680</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>680</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>680</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Please input your judgement:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Check">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>680</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Check</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
